<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>621</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>PID Control</string>
   </property>
   <widget class="QPushButton" name="button_hover">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Hover</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_land">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Land</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_circle">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Circle (20s)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_save_pid">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="text_drone_state">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Drone State:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>621</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string>Neural Control</string>
   </property>
   <widget class="QPushButton" name="button_neural_start">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_neural_stop">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Path</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_path">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_neural_transfer">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Transfer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_neural_load">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_adjust">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Adjust</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_adjust">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.500000000000000</double>
    </property>
    <property name="maximum">
     <double>2.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.050000000000000</double>
    </property>
    <property name="value">
     <double>1.500000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Inference (ms)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_auto_land">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto-Land</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_land">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>25.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>20.100000000000001</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Land after (sec)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_record">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Record Exec.</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_save_record">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>240</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="button_on">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>On</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
